views/skus/create.erb

```diff
@@ -0,0 +1,86 @@
+<div class="container">
+  <form action='/skus' name="createSku" method='post' onsubmit="return validateForm()">
+    <fieldset>
+    <legend> New Project </legend>
+    <% if @error %>
+      <div class="alert alert-danger" align="left">
+        <strong>Invalid Input:</strong>
+         That SKU already exists!
+       </div>
+    <% end %>
+
+    <div class="form-group row">
+      <label class="col-md-offset-2 col-md-5 control-label col-md-offset-2">Name</label>
+        <div class="col-md-offset-2  col-md-5 inputGroupContainer col-md-offset-2">
+          <div class="input-group">
+            <span class="input-group-addon">
+              <i class="glyphicon glyphicon-book"></i>
+            </span>
+            <input name="name" placeholder="New SKU Name" class="form-control" type="text">
+          </div>
+        </div>
+     </div>
+
+     <div class="form-group row">
+       <label class="col-md-offset-2 col-md-5 control-label col-md-offset-2">Client</label>
+       <div class="col-md-offset-2  col-md-5 inputGroupContainer col-md-offset-2">
+         <div class="input-group">
+           <span class="input-group-addon">
+             <i class="glyphicon glyphicon-list"></i>
+           </span>
+           <select name="sku_id" class="form-control selectpicker" >
+             <% for @sku in @skus %>
```

Comment:
This would be correct? 
`   <% for @client in @clients %>
                <% if @client.active %>
                   <option value="<%= @sku.id %>"><%= @sku.name %></option>`